src/common/util/date.ts

```diff
@@ -24,6 +24,36 @@ export const getRelativeDate = (value: Date | string) => {
   return formatRelative(date, new Date())
 }
 
+/**
+ * Gets the duration in time from the specified date until now
+ * @param date Date to get the interval from
+ */
+export const getDurationUntilNow = (date: Date) => {
+  return intervalToDuration({ start: date, end: new Date() })
+}
+
+/**
+ * Formats the given duration into a `{duration} {time} ago` string
+ * @param duration A date-fns `Duration` object to format as a string
+ */
+export const formatDuration = (duration: Duration) => {
+  let formattedString = ''
+  if (duration.seconds != 0) {
+    formattedString = `${duration.seconds} seconds ago`
+  }
+  if (duration.minutes != 0) {
+    formattedString = `${duration.minutes} minutes ago`
+  }
+  if (duration.hours != 0) {
+    formattedString = `${duration.hours} hours ago`
+  }
+  if (duration.days != 0) {
+    formattedString = `${duration.days} days ago`
+  }
+
+  return formattedString
+}
```

Comment:
@dimitur2204 In the next iterations you might consider using `formatRelative` fn from `date-fns`

https://date-fns.org/v2.28.0/docs/formatRelative

It can provide relative time in many locales

```js
formatRelative(addDays(new Date(), -6), new Date(), { locale: 'bg' })
```

Reply:
Done in https://github.com/podkrepi-bg/frontend/pull/709/commits/f9d6300e738619a13847789b1f5b473401603fb6